sdk/src/vote_program.rs

```diff
@@ -37,6 +38,32 @@ impl Vote {
     }
 }
 
+#[derive(Serialize, Default, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct StoredVote {
+    pub slot_height: u64,
+    pub confirmation_count: u32,
+}
+
+impl StoredVote {
+    pub fn new(vote: &Vote) -> Self {
+        Self {
+            slot_height: vote.slot_height,
+            confirmation_count: 1,
+        }
+    }
+
+    // The number of slots for which this vote is locked
+    pub fn lockout(&self) -> u64 {
```

Comment:
if the struct is a lockout, this would be expiration() or something like that...

Reply:
What if we name the struct `Commitment` or something like that?  The Vec it's in is the Locktower, right? So it's a tower of what? "Lock" isn't a very good option in this language.

Reply:
Thinking more on it `Lockout::expiration` sounds nice. Let's see if I can go 24 hours without renaming it.

Reply:
Was I too late or did you actually like Commitment better?

Reply:
I just changed it back to Lockout again xD

Reply:
Can you do that 5 more times to help make the rest of us feel more productive?

Reply:
I have a bad feeling that might be an underestimate xD